apps/els_lsp/src/els_definition_provider.erl

```diff
@@ -45,8 +45,14 @@ goto_definition(_Uri, []) ->
     null;
 goto_definition(Uri, [POI | Rest]) ->
     case els_code_navigation:goto_definition(Uri, POI) of
-        {ok, DefUri, #{range := Range}} ->
-            #{uri => DefUri, range => els_protocol:range(Range)};
+        {ok, Definitions} ->
```

Comment:
This now returns a list of maps, so the signature needs to be updated.

Reply:
You can run `rebar3 dialyzer` locally to run this type of checks without waiting for CI.